02-express-tutorial/answers/06-basic-json.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+const { products } = require('./data');
+
+
+app.get('/', (req, res) => {
+  res.json(products);
+});
+
+app.listen(port, () => {
```

Comment:
I like that you changed your port to use a variable instead of hard-coding it. Is there a way we can make the port change without needing to change any lines of code to make it happen 🤔

Reply:
Yes,  we can use process.env.PORT environment variable to set the port dynamically. Basically, if PORT env variable is set it will use that value as the port number overwise to make the 3000 or 5000, 8080, wtv port you want by default.